### ISSUE TITLE
when run auto_pruners_torch.py with save file flag, the log print out "Layer: conv1  Sparsity: 0.0000",  infact the sparsity is 0.3

### ISSUE THREAD

**Issue by weiqiangshen** [microsoft/nni#2764]
**Environment**:ubuntu 18.04
- NNI version: 1.7
- NNI mode (local|remote|pai): local
- Client OS: ubuntu 18.04
- Server OS (for remote mode only):
- Python version: 3.6
- PyTorch/TensorFlow version:1.5.1+cu101
- Is conda/virtualenv/venv used?: no
- Is running in Docker?: no

**Log message**:

Evaluation result (masked model): 0.9833
[08/04/2020, 12:00:00 PM] INFO (nni.compression.torch.compressor) Layer: conv1  Sparsity: 0.0000
[08/04/2020, 12:00:00 PM] INFO (nni.compression.torch.compressor) Layer: conv2  Sparsity: 0.0000
[08/04/2020, 12:00:00 PM] INFO (nni.compression.torch.compressor) Model state_dict saved to ./model_masked.pth
[08/04/2020, 12:00:00 PM] INFO (nni.compression.torch.compressor) Mask dict saved to ./mask.pth

 - nnimanager.log: 
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**What issue meet, what's expected?**:

**How to reproduce it?**: 
python3 auto_pruners_torch.py
**Additional information**:

**Comment by suiguoxin** [comment]
This issue is fixed in PR https://github.com/microsoft/nni/pull/2736. 

Thanks for your comments @weiqiangshen , please fetch the latest code from the master branch.

**Comment by scarlett2018** [comment]
@weiqiangshen - as a reference, this fix will be released on the upcoming release: https://github.com/microsoft/nni/issues/2608 (current target release date is Aug 17).

**Comment by kvartet** [comment]
I’m closing this issue as we have fixed the problem you raised, please feel free to reopen if you are still seeing it an active issue.
